qiskit_experiments/experiment_data.py

```diff
@@ -91,32 +55,6 @@ def experiment(self):
         """
         return self._experiment
 
-    @property
-    def experiment_type(self) -> str:
-        """Return the experiment type."""
-        return self._type
-
-    @property
-    def experiment_id(self) -> str:
-        """Return the experiment id."""
-        return self._id
-
-    def metadata(self) -> Dict:
-        """Return experiment metadata.
-        Returns:
-            Experiment metadata.
-        """
-        return self._metadata
-
-    @property
-    def job_ids(self) -> List[str]:
-        """Return experiment job IDs.
-
-        Returns:
-            IDs of jobs submitted for this experiment.
-        """
-        return list(self._jobs.keys())
-
     @property
     def completion_times(self) -> Dict[str, datetime]:
```

Comment:
Does `completion_times` belong to `ExperimentData` or `StoredData`?  @jyu00 @eggerdj

Reply:
Completion time is not stored in the DB since it's calculated dynamically, so it should belong here in `ExperimentData`. However, this method should also be fixed since `time_per_step()` is an IBMQ only attribute.

Reply:
Is there a reason this method is in `ExperimentData` instead of `DbExperimentDataV1`?

Reply:
@chriseclectic, here's @jyu's answer to this question: https://github.com/Qiskit/qiskit-experiments/pull/115#discussion_r665391695 (#162 is related)